source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,135 @@
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+## Home Assistant OS Snapshots
+
+Snapshots are a backup of your Home Assistant and addon data and configuration which are stored in a compressed archive file (.tar). Snapshots are made from the Supervisor Snapshot panel. There is also a service available which allows you to trigger the creation of a snapshot from an automation. Backups are stored in the /backup directory.
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* addons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+A partial snapshot consists of any number of the above default directories and installed add-ons.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not. Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+5. Click "Create" to begin the snapshot.
+
+### Restoring a Snapshot on a new install
+
+You can make use of snapshots which you have copied off of a previous install to restore to a new installation during the onboarding process. Follow the link at the bottom of the account creation page to upload your snapshot from the previous installation.
+
+For restoring a snapshot at any other time, vist the Supervisor Snapshot panel in your UI and use the following steps:
+
+1. Select "Upload Snapshot" from the icon in the upper right of the page.
+2. Click on the folder icon to navigate to your snapshot .tar file and select it.
+
+When the upload is completed, you will be presented with the snapshot restore dialog for restoring it, and can then choose to restore in full or in part by manually selecting individual items.
+
+If the snapshot you are uploading is more than 1GB in size, it can be faster and more efficient to make use of the Samba add-on in order to transfer files to the `/backup` directory.
+
+The length of time it takes to create or restore snapshots will depend on how much you have to compress or decompress. 
+
+If you're looking to slim down your snapshots, check if your configuration contains a large database file (`home-assistant_v2.db` in the config directory). See the [`recorder`](https://www.home-assistant.io/components/recorder/) integration page for options to keep your database data down to a size that won't cause issues. Note the keep days, purge interval, and include/exclude options.
+
+When the restore is complete, you will lose connection to the UI and it will return just as though you had restarted Home Assistant.
```

Comment:
```suggestion
When the restore is complete, Home Assistant will restart to apply the new settings. You will lose the connection to the UI and it will return once the restart is completed.
```